PlayByPostService that provides a place to write campaign notes

gludington opened the pull request:
…aign notes on a PBP thread.

- needs scss fixes
- unlimitedStorage as discussed on threads
- tinymce instead of textarea if storage increased?

emfmesquita commented on src/contentscript/extensioncontentscript.js:
```diff
@@ -79,6 +80,9 @@ ConfigStorageService.getConfig().then((config: Configuration) => {
     if (config[Opt.CampaignCharactersFolders]) CampaignCharactersService.init();
 
+    // change my characters page
+    if (config[Opt.MyCharactersFolders]) PlayByPostService.init();
```
Maybe on the final version create a new config option to turn on/off.

emfmesquita commented on src/services/storage/PlayByPostStorageService.js:
```diff
@@ -0,0 +1,31 @@
+import PlayByPostData from '../../data/PlayByPostData';
+import Q from "./Q";
+import StorageService from "./StorageService";
```
Probably later today or tomorrow I will send a commit with a couple of changes on storage. You wont have merging problems, just need to change StorageService to SyncStorageService (uses chrome.storage.sync) or LocalStorageService (uses chrome.storage.local).

gludington commented:
Updated the PR with a better options panel, changed to LocalStorageService.  Left a method in there that theoretically would let the service go back and forth between storage mechanisms, but I agree with you that the size restrictions are too problematic and that notes should state Local.
